requirement_data.js

```diff
@@ -474,9 +459,9 @@ const requirement_data = {
             new Requirement(5, [[Moves.LevelSlam]]), // 1 bunch in Ballroom
             new Requirement(10, [[Moves.Feather]]), // 1 balloon in LowerCave
             new Requirement(5, [[Moves.Mini]]), // 1 bunch in TrashCan
-            new Requirement(15, [[Moves.LevelSlam, Moves.Monkeyport]]), // 1 bunch, 2 bunches in MuseumBehindGlass
+            new Requirement(15, [[Moves.LevelSlam, Moves.Monkeyport, Moves.Diddy]]), // 1 bunch, 2 bunches in MuseumBehindGlass
```

Comment:
What is 'Diddy' here? Is this just encoding that you needed diddy to slam the switch? I feel like there's got to be other requirements that work like this...